### DISCUSSION TITLE
We have to think about our large assets in production 🏗️

### DISCUSSION THREAD

**Discussion by Im-Boss-13** [DawnBee/college-medicine-app#127] [Q&A] [answered]
Any ideas @DawnBee ?

**Comment by DawnBee** [comment, marked as answer]
> Any ideas @DawnBee ?

Agreed, I'm kindof worried as well. But I found about Git LFS (Large File Storage) it's an open-source extension of Git that allows tracking of large files.

Then in 'production' we might have to use Amazon's S3 or Bitbuckets LFS.
